lib/config/config-file.js

```diff
@@ -47,6 +51,10 @@ const loadConfig = () => {
     ]
   }
 
+  if (configFile) {
+    cosmiconfigOptions.searchPlaces.push(configFile)
```

Comment:
I merged this PR but then I realized this `configFile` should be added at the beginning of the `searchPlaces` array, not at the end. Otherwise, if you have a `.solhint.json` file but for some reason want to manually specify a different file, you won't be able to do it.